keyboards/octagon/octagon.h

```diff
@@ -0,0 +1,10 @@
+#ifndef OCTAGON_H
+#define OCTAGON_H
+
+#include "quantum.h"
+
+#ifdef KEYBOARD_octagon_v2
+    #include "octagon_v2.h"
```

Comment:
So the folder name and file name should really match here, so this should really be `v2.h` (and `v2.c`) to match the `v2/` folder.